### ISSUE TITLE
Multiple instances of the same 'instance'

### ISSUE THREAD

**Issue by a-person1908** [modrinth/theseus#302]
### Is your suggested enhancement related to a problem? Please describe.

I Think you should be able to play multiple instances of an 'instance' for example how the Official Minecraft Launcher allows you to launch multiple instances of the same profile.

### Describe the solution you'd like

I think adding a button to launch another instance would be a good solution.

### Describe alternatives you've considered

An alternative would be to have the play button remain and add a stop button dropdown, Though that would have a few problems such as knowing wich instance was wich.

### Additional context

![image](https://github.com/modrinth/theseus/assets/137985347/aca1a681-04ca-4373-902e-3ce1335b542f)
![Screenshot 2023-06-30 175002](https://github.com/modrinth/theseus/assets/137985347/37133015-ccf6-49a1-a6ed-a060d739cda4)
![Screenshot 2023-06-30 174227](https://github.com/modrinth/theseus/assets/137985347/48d8df80-ee17-4595-a641-3ef9fa6f1601)

**Comment by Thinkseal** [comment]
The main problem with this is that you run you very high risk of breaking something if you write/read from the same spot and since modrinth wants to reduce that risk they would need to find some way of making them isolated from each other while using the same data

**Comment by ItzTech-Official** [comment]
please man cuz I aint waiting on a hypixel alpha server queue 😭

**Comment by Sellingbakedpotatoes** [comment]
yeah i agree with this, it was hella convinnent. Just make us sign a form saying we understand the consequences or somethin

**Comment by ZmaZes** [comment]
The reason all of these duplicate instances are showing up _now_ is because this _**was**_ a feature of modrinth up until about a week ago. You would simply reopen the modrinth app and launch your instance second time. Now there is a new error screen when you try to do this. As other users have mentioned, there is nothing preventing the app from just launching another instance. We know the risks. Every other launcher out there can do it, including the vanilla one. You can even use another launcher to bypass modrinth's arbitrary restriction and launch a modrinth profile a second time. Please, modrinth devs, review this and consider allowing us to continue launching multiple instances of the same profile. The fix doesn't have to be an official feature, just remove the error screen and let it work like before. As a datapack dev this is heartbreaking and will ultimately lead to me switching to a different launcher if this is not addressed quickly.

<img width="386" height="151" alt="The new error screen that does not need to be there" src="https://github.com/user-attachments/assets/b350ae29-6a61-4a57-ac9d-f1e8ac6031eb" />
